src/cli/podcast_commands.py

```
@@ -0,0 +1,459 @@
+        )
+
+        limit = args.limit or 50
+
+        if args.async_mode:
+            logger.info(f"Downloading up to {limit} episodes (async mode)")
+            result = asyncio.run(downloader.download_pending_async(limit=limit))
+        else:
+            logger.info(f"Downloading up to {limit} episodes")
+            result = downloader.download_pending(limit=limit)
+
+        print(f"\nDownload complete:")
```

Ruff (0.14.7)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)